volume/parser.lcow.go

```diff
@@ -0,0 +1,35 @@
+package volume
```

Comment:
Normally I believe underscores are used in go filenames instead of `.`.  I'm guessing this was done because `parser_linux.go` would imply a linux build tag (which we don't want).  Maybe `parser_lcow.go` and `parser_linux_volume.go` ?